src/main/java/com/softserve/messaging/service/KafkaConsumer.java

```diff
@@ -0,0 +1,40 @@
+package com.softserve.messaging.service;
+
+import com.softserve.messaging.dto.MailTemplateDto;
+import com.softserve.messaging.model.MessageHandler;
+import com.softserve.messaging.repository.MessageHandlerRepo;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.springframework.kafka.annotation.KafkaListener;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class KafkaConsumer {
```

Comment:
Kafka is tied to the transport layer (like controllers), it shouldn't be a service.

Reply:
done